packages/react/src/hooks/useAgent.ts

```diff
@@ -77,6 +79,11 @@ type AgentStateAvailable = AgentStateCommon & {
   state: 'listening' | 'thinking' | 'speaking';
   failureReasons: null;
 
+  /** The client's assigned identity, coming from the JWT token. */
+  identity: Participant['identity'];
```

Comment:
is it the agent's identity? or the end-user's identity? the comment seems to be indicating the latter. (i.e. `client`).

assuming it's the agent's, i'd recommend changing the comment to be clear.